src/debugger.lisp

```diff
@@ -72,32 +95,36 @@ object the debugger was entered with."
   "Prints a list of available restarts."
   (let ((restarts (compute-restarts condition)))
     (cond (restarts
-           (format stream "~&;; Available restarts:~%")
+           (format stream "Available restarts:")
            (loop with max-name-length = (restart-max-name-length restarts)
                  for i from 0
                  for restart in restarts
                  for report = (handler-case (princ-to-string restart)
                                 (error () "#<error while reporting restart>"))
                  for restart-name = (or (restart-name restart) "")
-                 do (format stream ";; ~2,' D: [~vA] ~A~%"
-                            i max-name-length restart-name report)))
-          (t (format stream "~&;; No available restarts.~%")))))
+                 do (format stream "~%~2,' D: [~vA] "
+                            i max-name-length restart-name)
+                    (handler-case
+                        (format stream "~@<~A~:>" restart)
+                      (error ()
+                        (format stream "#<error while reporting restart>")))))
```

Comment:
Why is this duplicated with L80 in the old code? We already have the `report` variable, is it unused now?

Reply:
Oh whoops, it should be gone now